src/Parser/YaccParser.mly

```diff
@@ -270,20 +460,20 @@ data_rec_rest
 /* ========================================================================= */
 
 def
-: KW_IMPLICIT TLID implicit_ty_args type_annot_opt
+:  KW_IMPLICIT TLID implicit_ty_args type_annot_opt
     { make (DImplicit($2, $3, $4)) }
-| KW_METHOD expr EQ expr { make (DMethod($2, $4)) }
```

Comment:
Methods were moved to `def_10`, which means that they may be marked as public now. Is it intentional?